package.json

```diff
@@ -23,10 +23,10 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "tsc-check": "tsc --noEmit",
-    "prettier": "prettier --config ./.prettierrc.json --write './**/*.{js,tsx,ts}'",
-    "prettier:check": "prettier --config ./.prettierrc.json --check './**/*.{js,tsx,ts}'",
-    "lint:check": "eslint 'src/**/*.{js,tsx,ts}' --max-warnings=0 --cache --cache-location /tmp/eslint-cache.json && npm run tsc-check && npm run prettier:check",
-    "lint:fix": "eslint 'src/**/*.{js,tsx,ts}' --fix"
+    "prettier": "prettier --config ./.prettierrc.json --write \"./**/*.{js,tsx,ts}\"",
+    "prettier:check": "prettier --config ./.prettierrc.json --check \"./**/*.{js,tsx,ts}\"",
+    "lint:check": "eslint \"src/**/*.{js,tsx,ts}\" --max-warnings=0 --cache --cache-location /tmp/eslint-cache.json && npm run tsc-check && npm run prettier:check",
+    "lint:fix": "eslint \"src/**/*.{js,tsx,ts}\" --fix"
   },
```

Comment:
This was required because I was getting errors like

> ESLint: 5.7.0.
> No files matching the pattern "./src/assets/scripts/**/*.js" were found.
> Please check for typing mistakes in the pattern.